src/test/kotlin/com/lykke/trade/volume/monitoring/service/entity/TradeVolumeCacheTest.kt

```diff
@@ -210,4 +213,39 @@ class TradeVolumeCacheTest {
         assertEquals(1, result.size)
         assertEquals(BigDecimal.valueOf(40), result.first().second)
     }
+
+    @Test
+    fun testInitialization() {
+        val date = Date()
+        val date1 = Date(date.time - 1)
+        val date2 = Date(date.time - 2)
+        val date3 = Date(date.time - 3)
+        val oldDate = Date(date.time - 300)
+        Mockito.`when`(loader.loadEvents())
+                .thenReturn(listOf(
+                        EventPersistenceData(1L, date, listOf(
```

Comment:
As i understand should `EventPersistenceData.date` == `TradeVolumePersistenceData.date` ? But here we have different dates?

Reply:
No. This date is date of event (ExecutionEvent.Header.timestamp)